dev-gap/gbnp/gbnp-1.0.5.ebuild

```diff
@@ -0,0 +1,28 @@
+# Copyright 1999-2024 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit gap-pkg
+
+DESCRIPTION="Compute Gröbner bases of noncommutative polynomials"
+SLOT="0"
+SRC_URI="https://github.com/gap-packages/${PN}/archive/v${PV}.tar.gz -> ${P}.tar.gz"
+
+LICENSE="LGPL-2.1+"
+KEYWORDS="~amd64"
+IUSE="examples "
```

Comment:
trailing whitespace after `examples`

Reply:
Fixed. Interestingly, this commit crashed `pkgdev commit` (hi @arthurzam)